src/PersonObjects/README.md

```diff
@@ -1,2 +1,16 @@
 Implementation of all Person-type objects, including but not limited to
 the "PlayerObject" and Sleeves.
+
+Person.ts:
+Person.ts serves as a foundational piece for modeling characters within the system, 
+providing a structure for defining their attributes, skills, and behavior.
+
+A Person contains the following main attibutes: 
```

Comment:
I would cut everything from here to the end out. You're just re-iterating stuff that exists in the code, and what you say here will inevitably become out-of-date.

What you added before this is a good start (or you can leave it at just that) for what to add to the README: an overall sense of design/purpose. Here is for the high-level overview, the details are for the individual files.